docs/getting-started/create-defend-policy-api.asciidoc

```diff
@@ -0,0 +1,813 @@
+                "behavior_protection": {
+                  "message": "",
+                  "enabled": true
+                }
+              },
+              "logging": {
+                "file": "info"
+              },
+              "antivirus_registration": {
+                "enabled": false
+              },
+              "attack_surface_reduction": {
+                "credential_hardening": {
+                  "enabled": true
+                }
+              }
+            },
+            "mac": {
+              "events": {
+                "process": true,
+                "file": true,
+                "network": true
+              },
+              "malware": {
+                "mode": "prevent",
+                "blocklist": true
+              },
+              "behavior_protection": {
+                "mode": "prevent",
+                "supported": true
+              },
+              "memory_protection": {
+                "mode": "prevent",
+                "supported": true
+              },
+              "popup": {
+                "malware": {
+                  "message": "",
+                  "enabled": true
+                },
+                "behavior_protection": {
+                  "message": "",
+                  "enabled": true
+                },
+                "memory_protection": {
+                  "message": "",
+                  "enabled": true
+                }
+              },
+              "logging": {
+                "file": "info"
+              }
+            },
+            "linux": {
+              "events": {
+                "process": true,
+                "file": true,
+                "network": true,
+                "session_data": false,
+                "tty_io": false
+              },
+              "malware": {
+                "mode": "prevent",
+                "blocklist": true
+              },
+              "behavior_protection": {
+                "mode": "prevent",
+                "supported": true
+              },
+              "memory_protection": {
+                "mode": "prevent",
+                "supported": true
+              },
+              "popup": {
+                "malware": {
+                  "message": "",
+                  "enabled": true
+                },
+                "behavior_protection": {
+                  "message": "",
+                  "enabled": true
+                },
+                "memory_protection": {
+                  "message": "",
+                  "enabled": true
+                }
+              },
+              "logging": {
+                "file": "info"
+              }
+            }
+          }
+        }
+      }
+    }
+  ]
+}' \
+  --compressed
+----
+<1> `<PACKAGE-POLICY-ID>` to be replaced
+<2> `<KIBANA-VERSION>` to be replaced
+<3> `<LATEST-ELASTIC-DEFEND-PACKAGE-VERSION>` to be replaced
+
+You can repeat steps 2 and 3 if you need to make more modifications to the {elastic-defend} policy.
```

Comment:
```suggestion
You can repeat steps 2 and 3 to make more modifications to the {elastic-defend} policy.
```

Reply:
This line feels buried under the lengthy code example, and I feel like it could get overlooked. WDYT about moving it all the way to the top of the page, in the intro, to let readers know from the beginning how they can further modify the policy? Something like:

"This is a three-step process involving the Fleet API. You can repeat steps 2 and 3 to make more modifications..."